source/uwp/Renderer/lib/AdaptiveChoiceSetInputRenderer.cpp

```diff
@@ -169,6 +169,28 @@ namespace winrt::AdaptiveCards::Rendering::Uwp::implementation
                     // Otherwise, leave all options unset
                     radioButton.IsChecked(IsChoiceSelected(values, input));
                 }
+
+                radioButton.PreviewKeyDown(
+                    [this, stackPanel](winrt::IInspectable const& sender, winrt::KeyRoutedEventArgs const& args) -> void
```

Comment:
@paulcam206, `sender` here should always be the radioButton of interest, right? RadioButton doesn't have any "children" per see that can bubble this event up, so we should be safe here?:)

Reply:
I went ahead and passed `radioButton` in the lambda capture so that we know this won't be an issue.